frontend/app/.server/express/server.ts

```diff
@@ -81,6 +81,10 @@ app.all('*', rrRequestHandler(viteDevServer));
 log.info('  ✓ registering global error handler');
 app.use(globalErrorHandler());
 
-log.info('Server initialization complete');
+log.info('Server initialization completed with runtime configuration: %o', {
+  client: Object.fromEntries(Object.entries(clientEnvironment).sort()),
+  server: Object.fromEntries(Object.entries(serverEnvironment).sort()),
```

Comment:
[Array.prototype.toSorted()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted)

```suggestion
  client: Object.fromEntries(Object.entries(clientEnvironment).toSorted()),
  server: Object.fromEntries(Object.entries(serverEnvironment).toSorted()),
```

Reply:
```
Property 'toSorted' does not exist on type '[string, string | boolean][]'.
```

I don't want to set the compiler lib to ES2023 for the client bundle, so I'll have to look at creating a `tsconfig.server.json` file for the server bundles.

Reply:
In this usage context it's good... for now. 😄